apps/desktop/src/lib/branch/BranchLaneContextMenu.svelte

```diff
@@ -52,6 +52,22 @@
 	function saveAndUnapply() {
 		branchController.saveAndUnapply(branch.id);
 	}
+
+	const isVirginLane = $derived(
+		branch.name.toLowerCase().includes('lane') && commits.length === 0 && branch.files?.length === 0
```

Comment:
This doesn't make sense as a pushed/unpushed check, how do we tell if all of the branches in a given lane are integrated. Presumably we could use that mechanism.